Data/DAS_Data_Convention.md

```diff
@@ -4,8 +4,8 @@ Note: Currently module 5 re-runs module 3 inside module 5. Needs to be updated.
 
```

Comment:
Can you remove the Note "Currently module 5 re-runs module 3 inside module 5. Needs to be updated." this should be converted to an issue if you think it's worth noting (not sure it is), else you'll just refactor naturally when migrating the functions of modules 3 and 5 ! 

Also add description of this dataset : 

```suggestion
DAS Data corresponds to the output of Damage summation module.
```